### ISSUE TITLE
CPU gets high rates almost over 30%

### ISSUE THREAD

**Issue by youngdeok** [flutter/flutter#52067]
I don't know why the flutter plugin consumes a lot of CPU power and memory.
I've checked all of the processes on my laptop and I found that it's coming from the IDE with flutter.
If I use the flutter code, It starts to get high CPU on Android Studio, VSCode, and IntelliJ IDEA.
However, when I use the Kotlin code only, It looks stable but flutter is not.

The Laptop:
Windows 10 / i7-9750H / 16GB Memory / 64bits
![Annotation 2020-03-06 115638](https://user-images.githubusercontent.com/510922/76039865-cb18e080-5fa1-11ea-986d-dfb5e56bca9d.jpg)

Flutter:
```shell
flutter doctor
[√] Flutter (Channel stable, v1.12.13+hotfix.8, on Microsoft Windows [Version 10.0.18363.657], locale en-AU)

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[√] Android Studio (version 3.6)
[√] IntelliJ IDEA Community Edition (version 2019.3)
[√] Connected device (2 available)
```

**Comment by youngdeok** [comment]
But, it's not happened on my Mac pro.

**Comment by VladyslavBondarenko** [comment]
Hi @youngdeok,
could you provide full `flutter doctor -v`? 
are you running android app?

**Comment by youngdeok** [comment]
Yep! It's still happening. The weird thing is the cpu rate is up and down like regular frequency. 
It's coming on my laptop. I'd better to check on my desktop pc too. 
![Annotation 2020-03-10 085414](https://user-images.githubusercontent.com/510922/76260924-fe62b480-62ac-11ea-9d7a-db84d7649117.jpg)

Here:
>  flutter doctor -v
[√] Flutter (Channel stable, v1.12.13+hotfix.8, on Microsoft Windows [Version 10.0.18363.657], locale en-AU)
    • Flutter version 1.12.13+hotfix.8 at C:\DevKits\flutter
    • Framework revision 0b8abb4724 (4 weeks ago), 2020-02-11 11:44:36 -0800
    • Engine revision e1e6ced81d
    • Dart version 2.7.0


[√] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at C:\DevKits\AndroidSdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = C:\DevKits\AndroidSdk
    • Java binary at: C:\DevKits\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)
    • All Android licenses accepted.

[√] Android Studio (version 3.6)
    • Android Studio at C:\DevKits\Android Studio
    • Flutter plugin version 44.0.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)

[√] IntelliJ IDEA Community Edition (version 2019.3)
    • IntelliJ at C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2019.3.3
    • Flutter plugin version 44.0.3
    • Dart plugin version 193.6494.35

[√] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 10 (API 29) (emulator)

• No issues found!

**Comment by youngdeok** [comment]
Now, I got different results on my desktop.
I've checked all of each process and I finally found it. McAfee LiveSafe is running on my laptop. After I turned Firewall menu off in this software, it's gone. If I turn it on again, I can see the same symptom. 
![Annotation 2020-03-10 091115](https://user-images.githubusercontent.com/510922/76262342-2a336980-62b0-11ea-8356-b04ce6974ce0.jpg)
I'd better uninstall the Mcafee anti-virus. 
Sorry about this. :(
Flutter is awesome! 
Thank you!
